src/App.tsx

```diff
@@ -98,26 +109,41 @@ export default function App(): JSX.Element {
     }
   }
 
-  async function initializeFromTrustedNode() {
+  async function fillCheckpointFromNode() {
     try {
-      setReqStatusInit({loading: "Initializing from trusted node"});
+      setReqStatusInit({loading: "Fetching checkpoint from trusted node"});
 
       const client = getClient(configLeve, {baseUrl: beaconApiUrl});
       const res = await client.beacon.getStateFinalityCheckpoints("head");
       const finalizedCheckpoint = res.data.finalized;
-      await initializeFromCheckpoint(finalizedCheckpoint);
+      setCheckpointStr(toCheckpointStr(finalizedCheckpoint));
+
+      // Hasn't load clint, just disable loader
+      setReqStatusInit({});
     } catch (e) {
       e.message = `Error initializing from trusted node: ${e.message}`;
       setReqStatusInit({error: e});
       console.error(e);
     }
   }
 
+  function deleteState() {
+    deleteSnapshot();
+    setReqStatusInit({});
+  }
+
   return (
     <>
-      <Header />
-
       <main>
+        <section className="hero">
+          <h1>Lodestar Eth2.0 light-client demo</h1>
```

Comment:
We should replace instances of "Eth2.0" (old name) with "Ethereum consensus" (new name anticipating post-merge).